docs/samples/kafka/setup.py

```diff
@@ -30,7 +30,7 @@
     python_requires='>=3.7',
     packages=find_packages("transformer"),
     install_requires=[
-        "kserve",
+        "kserve>=0.9.0",
```

Comment:
wondering if there are better way to set the version, otherwise we need to update many places every time for the release.